drivers/public_reports/app/models/public_reports/homeless_population.rb

```diff
@@ -39,18 +39,20 @@ def generate_publish_url
     end
 
     def publish!(content)
-      # This should:
-      # 1. Take the contents of html and push it up to S3
-      # 2. Populate the published_url field
-      # 3. Populate the embed_code field
-      self.class.transaction do
-        unpublish_similar
-        update(
-          html: content,
-          published_url: generate_publish_url, # NOTE this isn't used in this report
-          embed_code: generate_embed_code, # NOTE this isn't used in this report
-          state: :published,
-        )
+      unless published?
```

Comment:
How does this interact with `unpublished_similar?`

Reply:
So, I excluded it from calling that section if it's already published because there was some sort of weird interaction.  Since only one can be published at a time (currently) this seemed safe enough.